docs/chart_best_practices/rbac.md

```diff
@@ -12,7 +12,7 @@ RBAC resources are:
 
 ## YAML Configuration
 
-RBAC and ServiceAccount configuration should happen under separate keys. They are separate things. Splitting these two concepts out in the YAML disambiguates them and make this clearer.
+RBAC and ServiceAccount configuration should happen under separate keys. They are separate things. Splitting these two concepts out in the YAML disambiguation them and make this clearer.
```

Comment:
disambiguates is the correct word grammatically.

Reply:
I see